vanilla-framework/vanilla-framework
# Remove line-height from <select> element

## Done

Fixed bug where descenders in text in select elements was being cut off at the bottom in Safari / Chrome

## QA

- Pull code
- Run `./run serve --watch`
- http://0.0.0.0:8101/vanilla-framework/examples/base/forms/selects/
- Verify default text is no longer being chopped at the bottom

## Details

Fixes #1289
